skeleton/acp/main_module.php.tpl

```diff
@@ -19,7 +19,7 @@ class main_module
 
 	function main($id, $mode)
 	{
-		global $user, $template, $request, $config;
+		global $config, $request, $template, $user;
```

Comment:
Either way you're still using globals so it doesn't really matter much.

But the reason it's like this is because this extension is supposed to be recreating the Acme Demo extension.